app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -140,11 +140,11 @@ Vue.mixin({
             `ethereum:${address}`;
           break;
 
-        case 'CELO': // waiting : pending
+        case 'CELO':
         case 'cUSD':
           qr_string = value ?
-            `celo://wallet/pay?address=${address}&amount=${value}` :
-            `celo://wallet/pay?address=${address}`;
+            `celo://wallet/pay?address=${address}&amount=${value}&token=${token_name}` :
+            `celo://wallet/pay?address=${address}&token=${token_name}`;
```

Comment:
to clarify, if the `token` param is not given, the deeplink will default to cUSD, so you can choose whether or not you want to provide it or not. For CELO though, the param is necessary.

Reply:
Yup! thanks for clarifying that @medhak1